doc/classes/Curve3D.xml

```diff
@@ -112,6 +112,18 @@
 				If [param idx] is out of bounds it is truncated to the first or last vertex, and [param t] is ignored. If the curve has no points, the function sends an error to the console, and returns [code](0, 0, 0)[/code].
 			</description>
 		</method>
+		<method name="sample_backed_with_rotation" qualifiers="const">
```

Comment:
```suggestion
		<method name="sample_baked_with_rotation" qualifiers="const">
```
I guess it should be `baked`, not `backed`?

Reply:
right. Thanks!